client/src/components/History/CurrentHistory/HistoryPanel.vue

```diff
@@ -41,31 +42,23 @@ interface BackendFilterError {
     ValueError?: string;
 }
 
-const props = defineProps({
-    listOffset: {
-        type: Number,
-        default: 0,
-    },
-    history: {
-        type: Object as PropType<Record<string, any> & GetComponentPropTypes<typeof HistoryCounter>["history"]>,
-        required: true,
-    },
-    filter: {
-        type: String,
-        default: "",
-    },
-    canEditHistory: {
-        type: Boolean,
-        default: true,
-    },
-    shouldShowControls: {
-        type: Boolean,
-        default: true,
-    },
-    filterable: {
-        type: Boolean,
-        default: false,
-    },
+interface Props {
+    listOffset?: number;
+    history: Record<string, any> & GetComponentPropTypes<typeof HistoryCounter>["history"];
```

Comment:
Minor thing, but we can import `import { HistorySummary } from "@/api";` and replace the type here to make it more readable.

```suggestion
    history: HistorySummary;
```

Reply:
Done, thank you!